pkgs/development/python-modules/morphys/default.nix

```diff
@@ -0,0 +1,23 @@
+{ buildPythonPackage
+, fetchPypi
+, lib
+}:
+buildPythonPackage rec {
+  pname = "morphys";
+  version = "1.0";
+  format = "wheel";
+
+  src = fetchPypi {
```

Comment:
please fetch from github so we can avoid building from a wheel
https://github.com/mkalinski/morphys

Reply:
ok.

Reply:
Done